hepdata/ext/elasticsearch/api.py

```diff
@@ -383,7 +403,9 @@ def index_record_ids(record_ids, index=None):
         to_index.append(doc)
 
     if to_index:
-        es.bulk(index=index, body=to_index, refresh=True)
+        result = es.bulk(index=index, body=to_index, refresh=True)
+        if result['errors']:
+            log.error('Bulk insert failed: %s' % result)
```

Comment:
@GraemeWatt This change should hopefully make the reindexing error messages more obvious in Celery.